src/cmd/flux-mini.py

```diff
@@ -1374,6 +1374,8 @@ def main(self, args):
         for xargs in commands:
             total += len(self.cc_list(xargs))
 
+        if total == 0:
+            raise ValueError("no jobs provided for bulk submission")
```

Comment:
Without this check, when `total == 0`, a ZeroDivisionError is raised when printing the progress bar.

Reply:
hey james, it seems like there are three fixes in this one commit.  Could they be split up?  AFAICT, there's a typo fix, there's the `total == 0` fix, and the help message fix.

Reply:
I thought someone might comment on that. I made it two commits, because I think the typo falls under the help message fix (it's part of the bulksubmit help message)